tests/query_tests.rs

```diff
@@ -214,7 +214,8 @@ fn partial_index() {
                 "new_index",
                 &ValueIndexConfiguration::new(
                     QueryLanguage::JSON,
-                    r#"{"WHAT": [[".id"]], "WHERE": ["=", [".someField"], "someValue"]}"#
+                    r#"{"WHAT": [[".id"]], "WHERE": ["=", [".someField"], "someValue"]}"#,
+                    ""
```

Comment:
This is the only partial index test, maybe we could duplicate it and use the newly added `where` argument in the second test?
Something like
`
db.create_index(
  "new_index",
  &ValueIndexConfiguration::new(
    QueryLanguage::JSON,
    r#"[[".someField"]]"#,
    r#"[["=", [".someField"], "someValue"]]"#
  ),
)
.unwrap()
`
Not sure how it is written for JSON indexes :-)